tests/test1.yaml

```
@@ -333,6 +333,12 @@ mcp23s17:
     deviceaddress: 1
 
 sensor:
+  - platform: xgzp68xx
+    temperature:
+      name: "Pressure Temperature"
+    pressure:
+      name: "Differential pressure"
```

Quotes are not needed around simple strings.
```suggestion
      name: Pressure Temperature
    pressure:
      name: Differential pressure
```